warehouse/query-core/src/main/java/datawave/query/jexl/functions/EvaluationPhaseFilterFunctionsDescriptor.java

```diff
@@ -41,7 +41,7 @@
         private static final ImmutableSet<String> dateBetweenFunctions = ImmutableSet.of("betweenDates", "betweenLoadDates");
         private static final String MATCHCOUNTOF = "matchesAtLeastCountOf";
         private static final String TIMEFUNCTION = "timeFunction";
-        
+        private static final String OPTIONS_FUNCTION = "options";
```

Comment:
Need to update getFields functions to return an empty list for the options function

Reply:
The arguments for filter:options are all of StringLiteral type so the call to getIdentifiers (called when you get fields) correctly returns an empty collection.